### ISSUE TITLE
How is this module not redundant with twig_tweak?

### ISSUE THREAD

**Issue by brookeheaton** [BurdaPraha/DrupalTwigFood#6]

**Comment by landsman** [comment]
Can be, but we want be able to add new features in quick time. It's reason why we using own module for that. And I was want to try create my first module for Drupal 8. Do you have some suggestions @brookeheaton?

**Comment by brookeheaton** [comment]
If you are registered on Drupal.org, you should be able to start a project by going to https://www.drupal.org/project/user and add_project - your project will begin as a sandbox, but if there is enough usage and it is secure, it can be a contributed module. Once your module is up, you may want to connect with the twig_tweak team to see if any of your functionality can be merged in or if they are distinct enough to warrant being separate and distinct modules with distinct aims.

**Comment by landsman** [comment]
@brookeheaton Yes, I am registred: https://www.drupal.org/u/landsman and after I be done with this module (issues) I will publish it as drupal project. How do you found this module? What do you need to solve in your project?
